bin/maintenance/build-wheel.py

```diff
@@ -174,6 +174,28 @@ def patch_indico_version(add_version_suffix):
             f.write(old_content)
 
 
+@contextmanager
+def patch_plugin_version(add_version_suffix):
```

Comment:
Any chance we can avoid duplicating so much code? I guess the changes between the two version patching functions are minimal...